aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -133,6 +133,21 @@
     "!aviary run_mission -h"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`input_deck` is the path to vehicle input deck .csv file.\n",
+    "`-o` or `--outdir` is the directory to write outputs. The default is current directory.\n",
+    "`--optimizer` is the name of the optimizer. The default is `SNOPT`.\n",
+    "`--shooting` indicates that the integration method is shooting method instead of collocation scheme. The default is collocation.\n",
+    "`--phase_info` is the path to phase info file. If it is missing, it depends on the integration method (collocation or shooting) and on the mission method (settings:equations+of_motion with value of `2DOF` or `height_energy`) which is defined in the .csv input file.\n",
+    "`--n2` indicates whether the user wants to generate an n2 diagram after the analysis. The default is `False`. TODO: Currently, it does not have any impact. It should be removed.\n",
```

Comment:
```suggestion
```

Reply:
this is incorrect, the n2 flag is used to trigger this code block in `run_level_1` in methods_for_level1.py

Reply:
I tried to run with `--n2` on and off. I don't see any difference.

Reply:
This PR removed the portion of code in methods_for_level1 that did this:
https://github.com/OpenMDAO/Aviary/pull/548#discussion_r1788309676

Reply:
I tested it to confirm that option `--n2` has no impact and then removed the portion of code in methods_for_level1. Can you tell me what I should expect when we run `aviary run_mission` with and without `--n2` option?